docs/frontmatter.md

```diff
@@ -658,7 +658,25 @@ String values for licenses should be a valid “Identifier” string from the [S
     - `AGPL`
 ```
 
-By using the correct SPDX Identifier, your website will automatically use the appropriate icon for the license and link to the license definition.
+By using the correct SPDX Identifier, your website will automatically use the appropriate icon for the license and link to the license definition.  The simplest and most common example is something like:
+
+```yaml
+license: CC-BY-4.0
+```
+
+### Nonstandard licenses
```

Comment:
Something that we are seeing a lot of in the publishing world. Exciting to be able to support these in MyST.